tests/test_visitors/test_ast/test_comparisons/test_multiple_in.py

```diff
@@ -14,17 +14,19 @@
 ternary_with_multiple_in = 'ternary = 0 if {0} in {1} in {2} else 1'
 while_construct = 'while {0} in {1}: ...'
 while_with_multiple_in = 'while {0} in {1} in {2}: ...'
+if_with_refactored_in = 'if x_coord in line and line in square: ...'
 
 
 @pytest.mark.parametrize('code', [
     if_without_multiple_in_comparisons,
     ternary,
     while_construct,
+    if_with_refactored_in,
 ])
 @pytest.mark.parametrize('comparators', [
-    ('6', '6'),
-    ('6', [True]),
-    ('a', ['a', 'b']),
+    ('x', '6'),
```

Comment:
Can please rename this variable to follow minimum length rule? For example, name it `x_coord`